documentation/service-rules/school-admins.md

```diff
@@ -117,11 +137,246 @@ We ask this question so we can get a better understanding of when the ECT is sta
 
 The school start date is important because:
 
-* it changes what appropriate body the ECT can be supported by
-* it changes what lead provider and delivery partner combination the ECT can be trained by
 * it informs lead providers of when the training should start for the ECT
 * it alters what funding the ECT might be eligible for
 
 You can read more about why we chose to add this question [in the design history entry here](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/ects-start-date/).
 
+We’ll update this later with more information on the validation and rules for the start date that can be given. 
+
+### Giving an ECT’s working pattern
+
+We always ask for the working pattern for an ECT so:
+
+* the ECT can be onboarded to the correct kind of training with a lead provider
+* the lead provider or delivery partner can enquire more with the school for part-time ECTs, to make sure they have all the information they need and are fully supported
+* we better understand the kinds of ECTs undergoing training 
+
+This question is new for Register early career teachers, so the working pattern field will not be populated for migrated records.
+
+You can read more about why we chose to add this question [in the design history entry here](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/asking-schools-whether-an-ect-is-full-or-part-time/).
+
+### Generating an expected training start year
+
+The expected training start year is generated from when the ECT is registered and when the school reports the ECT is starting. This is because if an ECT had started at a school in January 2024, which would be in the 2023 to 2024 academic year, but they’re not registered by school till September 2024, they wouldn’t be starting with a lead provider for training until the 2024 to 2025 academic year. Essentially, training can’t be expected to start until they’re registered with DfE and have their details passed to a lead provider. 
```

Comment:
```suggestion
The expected training start year is generated from when the ECT is registered and when the school reports the ECT is starting. This is because if an ECT had started at a school in January 2024, which would be in the 2023 to 2024 academic year, but they’re only registered by the school in September 2024, they wouldn’t be starting with a lead provider for training until the 2024 to 2025 academic year. Essentially, training can’t be expected to start until they’re registered with DfE and have their details passed to a lead provider. 
```